src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -14,7 +19,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), build.props))\build.props" />
 
   <ItemGroup>
-    <PackageReference Include="CommandLineParser" Version="2.2.1" />
+    <PackageReference Include="CommandLineParser" Version="2.5.0" />
```

Comment:
This was required to test the change in docker; the multitool breaks in docker using 2.2.1 due to https://github.com/commandlineparser/commandline/issues/153